include/boost/geometry/extensions/gis/projections/impl/pj_init.hpp

```diff
@@ -204,10 +206,23 @@ inline parameters pj_init(R const& arguments, bool use_defaults = true)
 
     if (! s.empty())
     {
-        // TODO: IMPLEMENT SPLIT
-        pin.to_meter = atof(s.c_str());
+        std::size_t const pos = s.find('/');
+        if (pos == std::string::npos)
+        {
+            pin.to_meter = lexical_cast<double>(s.c_str());
+        }
+        else
+        {
+            double const numerator = lexical_cast<double>(s.substr(0, divisor)), 
+                         denominator = lexical_cast<double>(s.substr(divisor + 1));
+            if (numerator == 0.0 || denominator == 0.0)
```

Comment:
OK, I'll try to remember but I'll probably forget a lot just because it is not my habit. Why do you prefer it that way?